packages/perseus-editor/src/widgets/phet-sim-editor.tsx

```diff
@@ -1,45 +1,55 @@
 /* eslint-disable @khanacademy/ts-no-error-suppressions */
 /* eslint-disable react/sort-comp */
-import {Changeable, EditorJsonify} from "@khanacademy/perseus";
+import {LabeledTextField} from "@khanacademy/wonder-blocks-form";
 import * as React from "react";
 
-import BlurInput from "../components/blur-input";
+import type {PerseusPhetSimWidgetOptions} from "@khanacademy/perseus/src/perseus-types";
 
-type PhetSimEditorProps = any;
+type Options = {
+    url: PerseusPhetSimWidgetOptions["url"];
+    description: PerseusPhetSimWidgetOptions["description"];
+};
```

Comment:
Two questions about this!

1. Is `Options` an okay name for this instead of `DefaultProps`? I want to use it as the return type of `this.serialize()` function which makes `DefaultProps` look like a bit of a misnomer.
2. Is this an okay way of setting the type for default props and serialize? If we just import `PerseusPhetSimWidgetOptions`, we get `static` as well, so this was our best strategy.

Reply:
1. The value returned by `serialize()` should really be a [`PerseusPhetSimWidgetOptions `](https://github.com/Khan/perseus/blob/phet-widget/packages/perseus/src/perseus-types.ts#L1544).  This data will be what's passed back to the `phet-sim`'s `transform` function and ultimately [provided](https://github.com/Khan/perseus/blob/phet-widget/packages/perseus/src/widgets/phet-sim.tsx#L20...L21) to the `PhetSim` React component.

I think it's fair to have a separate `DefaultProps` type which represents the default props for this editor widget. This could differ from the widget options and it is conceptually a very different type.

2. Yes, this is the conventional way we've defined default props (by using indexed properties into `Props`). As I said above, I would keep the `PerseusPhetSimWidgetOptions` type separate from `DefaultProps`. 

Is there a reason that `static` was added to `PerseusPhetSimWidgetOptions`? I don't think it's needed.

Reply:
Huh! I guess I could leave `static` out. When I was first making the PerseusPhetSimWidgetOptions, I was operating off the iframe, and iframe and some other widgets had this at the end of their widget options (which did not see a use for, so I thought it was for something I did not understand):
```
// Always false
static: boolean;
```
But I'm now seeing that some of the other widgets don't have a `static`, so I guess it's not a hard and fast rule! I'll go ahead and remove `static` and change the `serialize()` return type to `PerseusPhetSimWidgetOptions`. Thank you so much!

P.S. Do you know why `static` is on some widgets but not others?

Reply:
The individual widget options should never have `static` on them. It is already [defined](https://github.com/Khan/perseus/blob/f083200340545f41275f0696dbfc967f45028b0c/packages/perseus/src/types.ts#L582) in the base `WidgetProps<T>` type which is what all widgets [receive](https://github.com/Khan/perseus/blob/f083200340545f41275f0696dbfc967f45028b0c/packages/perseus/src/renderer.tsx#L658) from the renderer.